sync-zones-pipeline.yml

```diff
@@ -0,0 +1,141 @@
+# Docs:
+# https://aka.ms/yaml
+
+name: Azure Public DNS zone to Private DNS zone Sync up
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+  paths:
+    include:
+      - scripts/sync-to-private-dns-zone.sh
+      - sync-zones-pipeline.yml
+
+pr:
+  branches:
+    include:
+      - '*'
```

Comment:
Not a big deal but should we bother running this on PRs? I think it would be better to just run the script on master - unless you have it so that it runs but doesn't do anything on PRs?

Reply:
make sense, its going to run the script so no point running it on the PR, have removed it now.